R/explain.R

```diff
@@ -290,24 +319,32 @@ explain <- function(model,
                     x_explain,
                     x_train,
                     approach,
+                    paired_shap_sampling = TRUE,
                     prediction_zero,
-                    n_combinations = NULL,
+                    max_n_coalitions = NULL,
+                    adaptive = TRUE,
                     group = NULL,
-                    n_samples = 1e3,
-                    n_batches = NULL,
+                    n_MC_samples = 1e3,
                     seed = 1,
                     keep_samp_for_vS = FALSE,
                     predict_model = NULL,
                     get_model_specs = NULL,
                     MSEv_uniform_comb_weights = TRUE,
-                    timing = TRUE,
                     verbose = 0,
```

Comment:
I would suggest indicating all three verbose possibilities (i.e., `verbose = c(0, 1, 2)` and then using ` verbose = match.arg(verbose)` within the code to test that one and only one of these three integers is passed.

Reply:
The functionality of verbose is not changed.